URBNSwiftAlert/Classes/AlertStyler.swift

```diff
@@ -73,6 +73,16 @@ extension AlertStyler {
          * Corner radius of the alert view itself
          */
         public var cornerRadius: CGFloat = 0.0
+
+        /**
+         * Separator color between the title/message and button, uses urbn_topBorder on buttonSV
+         */
+        public var separatorColor: UIColor = .clear
+
+        /**
+         * Separator width betwenn the title/message and button, uses urbn_topBorder on buttonSV
+         */
+        public var separatorWidth: Int = 0
```

Comment:
Is this pixel or point? Should specify in the variable name

Reply:
I would be a bit more explicit with the naming of these. Maybe `buttonContainerSeperator....`.

Reply:
Pixel, will update